database/database.go

```diff
@@ -0,0 +1,26 @@
+package database
+
+import (
+	"csi-accounts/internal/models"
+	"log"
+
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+var DB *gorm.DB
+
+func ConnectDatabase() {
+	dsn := "host=localhost user=postgres password=IIMB@1802 dbname=csi_rbac_system port=5432 sslmode=disable TimeZone=Asia/Kolkata"
```

Comment:
database and migration file are present in initializers folder, so no need of rewrite. Also database credentials put in env.